plugins/org.eclipse.fordiac.ide.comgeneration/src/org/eclipse/fordiac/ide/comgeneration/plugin/messages.properties

```
@@ -20,5 +20,5 @@ CommFBGenerator_NoStartFBInResource= No start FB in resource {0}!
 
 ProtocolSelector_NoConnectionAvailableFor= No connection available
 
-CommGenerator_FBTypeEntryNotFound=FB type entry '{0}' not found!
+CommGenerator_FBTypeEntryNotFound=FB type entry {0} not found!
```

I believe using proper quotes would be more readable:
```suggestion
CommGenerator_FBTypeEntryNotFound=FB type entry ''{0}'' not found!
```

---

done